stl/src/locale.cpp

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-// class locale member functions

```

Comment:
huh?

Reply:
I simply missed one https://github.com/microsoft/STL/pull/2074#discussion_r674914034. We can have it both ways till a maintainer chimes in? 😅